src/Command/PermissionsMakeAdminCommand.php

```diff
@@ -51,7 +53,26 @@ protected function configure(): void
     protected function execute(InputInterface $input, OutputInterface $output): int
     {
         $io = new SymfonyStyle($input, $output);
+
         $discordUserId = $input->getArgument('discord_user_id');
+        if (null === $discordUserId) {
+            /** @var UserEntity[] $allUsers */
+            $allUsers = $this->userEntityRepository->findAll();
```

Comment:
ChoiceQuestion throws an exception when this array is empty. We should check if there are users here and abort command if not.